models/clipper/polygon/clipper_v2_polygon_trades.sql

```diff
@@ -16,22 +16,21 @@ WITH event_data as (
         evt_block_time AS block_time,
         evt_block_number as block_number,
         recipient as taker,
-        '' AS maker,
+        CAST(NULL AS VARBINARY) AS maker,
         CAST(inAmount AS DECIMAL(38,0)) AS token_sold_amount_raw,
         CAST(outAmount AS DECIMAL(38,0)) AS token_bought_amount_raw,
```

Comment:
i don't think we want to cast these? i would double check their data type in decoded table, then leave as-is if `uint256`, otherwise cast to `uint256` so downstream spells don't fail on mismatch of types